azimuth/modules/base_classes/module.py

```diff
@@ -164,6 +164,8 @@ def get_model(self):
         Raises:
             ValueError if no valid pipeline exists.
         """
+        if not self.can_load_model:
+            raise ValueError("This module cannot load the model. Set can_load_model to True.")
```

Comment:
Very good idea to catch it here! 👍 
Detail, but to be rigorous, it seems a [`ValueError`](https://docs.python.org/3/library/exceptions.html#ValueError) is not quite appropriate here:
> Raised when an operation or function receives an argument that has the right type but an inappropriate value

I think we can use a more generic [`RuntimeError`](https://docs.python.org/3/library/exceptions.html#RuntimeError):
> Raised when an error is detected that doesn’t fall in any of the other categories.

```suggestion
            raise RuntimeError("This module cannot load the model. Set can_load_model to True.")
```

Reply:
Perfect, I didn’t know what to put. 

It’s worth saying that this method (`can_load_model`) is not perfect, a module can call the prediction module with `.compute()` or `.compute_on_dataset_split()`, which will load the model without calling `get_model()`, the best example of this is perturbation testing. I’m not sure how we can make that more fail-proof. Perhaps I can add a TODO.